pom.xml

```diff
@@ -184,7 +184,6 @@
       <groupId>org.apache.apex</groupId>
       <artifactId>apex-engine</artifactId>
       <version>${apex.core.version}</version>
-      <scope>test</scope>
```

Comment:
Malhar code (except tests) should only depend on apex-api and apex-common, not engine internals.